src/test/java/spoon/reflect/visitor/CtVisitorTest.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (C) 2006-2015 INRIA and contributors
+ * Spoon - http://spoon.gforge.inria.fr/
+ *
+ * This software is governed by the CeCILL-C License under French law and
+ * abiding by the rules of distribution of free software. You can use, modify
+ * and/or redistribute the software under the terms of the CeCILL-C license as
+ * circulated by CEA, CNRS and INRIA at http://www.cecill.info.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the CeCILL-C License for more details.
+ *
+ * The fact that you are presently reading this means that you have had
+ * knowledge of the CeCILL-C license and that you accept its terms.
+ */
+
+package spoon.reflect.visitor;
+
+import org.junit.Test;
+import spoon.Launcher;
+import spoon.reflect.visitor.processors.CheckModelProcessor;
+
+public class CtVisitorTest {
+	@Test
+	public void testMethodsInVisitor() throws Exception {
+		final Launcher launcher = new Launcher();
+		launcher.getEnvironment().setNoClasspath(true);
+		launcher.addProcessor(new CheckModelProcessor(CtVisitor.class, message -> {
```

Comment:
what about launcher.addProcessor(new CheckThat(CtVisitor.class).hasRequiredScanners()) simply 

(the default strategy could be directly in the processor